.github/workflows/unit-test.yml

```
@@ -42,7 +42,7 @@ jobs:
 
       - name: Unit-Test
         run: |
-          GOOS=linux go test -v -race -coverpkg=./... -coverprofile=cover.out.tmp ./...
+          GOOS=linux TEST_DIRECTORY=./...  gotestsum --format pkgname -- -race -coverpkg=./... -coverprofile=cover.out.tmp ./...
```

try adding `go get gotest.tools/gotestsum` here to see if it fixes it
```suggestion
          go get gotest.tools/gotestsum
          GOOS=linux TEST_DIRECTORY=./...  gotestsum --format pkgname -- -race -coverpkg=./... -coverprofile=cover.out.tmp ./...
```

---

TIL `go get` is for modules, `go install` for CLIs